compose.go

```diff
@@ -9,11 +9,12 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"net/netip"
 	"reflect"
 	"unsafe"
 
+	"go4.org/netipx"
```

Comment:
woo IPX support! :P

Reply:
I considered the naming ambiguity a feature, actually. Sweet memories. :)